raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Reconsideration of Restoration Order
+(92) [1] Within days of the service of the environmental restoration order, a person who is served with an environmental restoration order may, by giving reasons in writing, request the Agency to reconsider that order; [2] Where a request has been made as provided for under subsection [1], the environmental restoration order shall continue to be effective until varied, suspended or withdrawn; [3] The Agency shall within days of receiving the request made in accordance with subsection [1], reconsider the environmental restoration order and notify in writing the person who made the request of its decision on the environmental restoration order; [4] The Agency may, after reconsidering the case, confirm, vary, suspend or withdraw the environmental restoration order; [5] The Agency may give the person who has requested a reconsideration of an environmental restoration an opportunity to be heard orally before a decision is made. 
+
+Action by an Environmental Inspector of the Agency in Case of Non Compliance with an Environmental Restoration Order
+(93) [1] Where a person on whom an environmental restoration order is served fails, the owner of a facility causing harm to the environment, or his or her agent neglects or refuses to take the action required by the order, an Environmental Inspector of the Agency and other officers and agents as may be required in the circumstances, may enter any land or premises under the control of the person on whom the environmental restoration order was served; and take all necessary action in respect of the activity to which that order relates so as to enforce that order an may be deemed fit; [2] Where the Agency exercised the power under subsection [1], it may recover as a civil debt, in the Environmental Court from the persons referred to in subsection [1], the expenses necessarily incurred by the Agency in the exercise of its power.
```

Comment:
Please replace:

* `served fails, the owner` with `served fails the owner`
* `order an may be deemed` with `order as may be deemed`
* `fit; [2]` with `fit. [2]`